protobufs/livekit_room.proto

```diff
@@ -59,11 +60,22 @@ service RoomService {
 
   // Update room metadata, will cause updates to be broadcasted to everyone in the room, Requires `roomAdmin`
   rpc UpdateRoomMetadata (UpdateRoomMetadataRequest) returns (Room);
+
+  // Create a room configuration. 
+  rpc CreateRoomConfiguration(CreateRoomConfigurationRequest) returns (RoomConfiguration);
```

Comment:
worth discussing: should these be APIs on room service? or Project settings in Cloud (and config in OSS)?

Reply:
if they are on room, we'd have to figure out permissions.. what token permissions would they require

Reply:
A "configuration service" would work too. I have no strong opinion either way. Does anybody else have any input? Either way, we'll need to sort out permissions.

Reply:
I would propose not to have a service for this initially. OSS would configure these in the config yaml. Cloud would store it per project.

Reply:
How would cloud allow defining these configurations?